app/forms/candidate_interface/completion_form.rb

```diff
@@ -0,0 +1,32 @@
+module CandidateInterface
+  class CompletionForm
+    include ActiveModel::Model
+
+    def self.build_from_application(attr_hash)
+      attr = attr_hash.keys.first
+
+      create_attr_accessor_from(attr)
+      create_validation_from(attr)
```

Comment:
This will not be required if we have a standardised field name
https://github.com/DFE-Digital/apply-for-teacher-training/pull/4549/files#r616524868